dallinger/utils.py

```diff
@@ -816,6 +822,14 @@ def map_locations_to(self, destination):
         raise NotImplementedError()
 
 
+def is_broken_symlink(path):
+    """Check if the given path is a broken symlink.
```

Comment:
Missing tests for new code decreases test coverage.

Reply:
Added now